vignettes/readr.Rmd

```diff
@@ -127,7 +127,7 @@ parse_number("$1,234")
 
 There are two parsers that will never be guessed: `col_skip()` and `col_factor()`. You will always need to supply these explicitly.
```

Comment:
Isn't it also true the `col_integer()` will not be guessed? I spotted this since we are working here in the immediate neighborhood.

Reply:
No, I don't think so. Here is where the guessing happens https://github.com/tidyverse/vroom/blob/0c2423eb3bba77b05ef9fda1830f1f798cc2fedd/src/guess_type.cc#L128-L146